server/test/integration/vitalsTest/handleHeartbeatTest.js

```diff
@@ -215,6 +215,17 @@ describe('vitals.js integration tests: handleHeartbeat', () => {
       sandbox.restore()
     })
 
+    it('should log a Sentry alert when the INS sensor value is less or equal to zero', async () => {
+      const response = await chai.request(server).post('/api/heartbeat').send({
+        coreid: radar_coreID,
+        data: `{"isINSZero":true,"doorMissedMsg":0,"doorMissedFrequently":false,"doorLastMessage":-1,"doorLowBatt":-1,"doorTampered":-1,"resetReason":"NONE","states":[]}`,
+        api_key: webhookAPIKey,
+      })
+
+      expect(response).to.have.status(200)
+      expect(helpers.logSentry).to.have.been.calledWith(`INS sensor is equal to or less than zero at ${testLocation1Id}`)
```

Comment:
In general, automated tests should have only a single assertion/expects-statement in them. This is so that earlier failures do not hide the results of later tests. 

In this case, if the response had a status code of 201 instead of 200, the test would fail immediately and you would have no way of knowing whether or Sentry was called with the correct message or not.

If you split this up into two tests, one that checks for the return value and the other that checks for the Sentry message, then you will have a better overall picture of the program when one, the other, or both fail.